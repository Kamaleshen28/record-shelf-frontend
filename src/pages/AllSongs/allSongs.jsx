import React from 'react'
import AllCards from '../../components/allCards/allCards'
import Header from '../../components/Header/header'

export default function AllSongs(){

    return(
        <div>
            <Header />
            <AllCards />
        </div>
    )
}