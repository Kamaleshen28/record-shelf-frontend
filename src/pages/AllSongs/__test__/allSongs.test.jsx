import { render} from '@testing-library/react';
import React from 'react';
import AllSongs from '../allSongs';
import { useNavigate } from 'react-router-dom';

jest.mock('react-router-dom');

describe('allSongs', () => {
    it('should render allSongs correctly', () => {
        const {asFragment} = render(<AllSongs />);
        expect(asFragment()).toMatchSnapshot();
    });
});