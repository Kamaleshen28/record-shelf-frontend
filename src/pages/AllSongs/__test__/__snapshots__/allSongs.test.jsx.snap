// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`allSongs should render allSongs correctly 1`] = `
<DocumentFragment>
  <div>
    <div
      class="header"
    >
      <p>
        My 
        <span
          class="record-header"
        >
          Record
        </span>
         Shelf
      </p>
    </div>
    <div
      class="body-dummy"
    >
      <div
        class="body-all-songs"
      >
        <div>
          <div
            class="body-header"
          >
            <p
              class="all-songs-text"
            >
              all songs
            </p>
            <img
              alt="Not Found"
              class="genre-icon"
              data-testid="genre-icon"
              src="icon-genre.svg"
            />
          </div>
          <div
            class="cards-container"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
